server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -40,3 +40,8 @@ db2SqlParser.size=%s not allowed. Size should be in GB.
 db2SqlParser.maxIntValue=%s not allowed. Allowed range is 0<k<=32767.
 db2SqlParser.currectQueryAccelaration=%s not valid. Must be of format %d%d%d%d.%d
 cobolParser.subSchemaNameLength=The length %s is not allowed. Allowed values are 16, 18.
+ContinuationLineTransformation.compilerDirectiveContinued=Compiler directives cannot be continued on another line. 
```

Comment:
@zimlu02  Please check these error messages.

Reply:
The first 2 miss "is", third has "is". Other from that looks OK. I would rather @zeibura to check this, pls.

Reply:
What do these error messages actually look like? Can you send me a screenshot of one of them?

Reply:
![image](https://user-images.githubusercontent.com/69206564/109638445-50e85100-7b4e-11eb-9c9c-a8d09d33f55d.png)
![image](https://user-images.githubusercontent.com/69206564/109638626-84c37680-7b4e-11eb-8bb5-dd8360abe860.png)

Reply:
I would suggest we do them like this

db2SqlParser.validValueMsg=Syntax error on %s expected %s
db2SqlParser.maxLength=Max length limit of %s bytes allowed for %s.
db2SqlParser.piecesize=Invalid size %s. Specify a size in KB, MB or GB.
db2SqlParser.size=Invalid size %s. Specify a size in GB.
db2SqlParser.maxIntValue=Invalid value %s. Allowed range is 0<k<=32767.
db2SqlParser.currectQueryAccelaration=Invalid format %s. Must be in format %d%d%d%d.%d

Reply:
cobolParser.subSchemaNameLength=Invalid length %s. Allowed values are 16, 18.
ContinuationLineTransformation.compilerDirectiveContinued=Compiler directives cannot be continued on another line. 
CompilerDirectivesTransformation.sequenceNumber=The first character of the sequence number must be numeric.
lexer.controlDirectiveWrongArgs=Invalid argument for CONTROL compiler directives.
lexer.langMissingEnterDirective=LanguageName missing for ENTER compiler directive.
lexer.titleCompilerDirective=Title statement missing for TITLE compiler directive.